bool detect_all
---
workshop_msgs/Detections detections
